Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "Miniscript: a subset of Bitcoin Script designed for analysis"
 license = "CC0-1.0"
 
 [features]
-fuzztarget = ["bitcoin/fuzztarget"]
+fuzztarget = []
```

Comment:
In f3c38b8cc04fed0a68f4d6074d8c30f6912d958f:

We should drop `fuzztarget` entirely.